packages/@mollitia/redis-storage/test/unit/module/breaker/sliding-count-breaker.spec.ts

```diff
@@ -0,0 +1,351 @@
+        failureRateThreshold: 60,
+        storage: {
+          use: true,
+          ttl: 2000
+        },
+        name: moduleName
+      };
+      const slidingCountBreaker = new Mollitia.SlidingCountBreaker(breakerData);
+      const slidingCountBreaker2 = new Mollitia.SlidingCountBreaker(breakerData);
+      const circuit = new Mollitia.Circuit({
+        options: {
+          modules: [slidingCountBreaker]
+        }
+      });
+      const circuit2 = new Mollitia.Circuit({
+        options: {
+          modules: [slidingCountBreaker2]
+        }
+      });
+      await slidingCountBreaker.clearState();
+      await expect(circuit.fn(failureAsync).execute('dummy')).rejects.toEqual('dummy');
+      await expect(circuit2.fn(successAsync).execute('dummy')).resolves.toEqual('dummy');
+      expect(slidingCountBreaker2.state).toEqual(Mollitia.BreakerState.CLOSED);
+      await delay(100);
+      let res = await storageAddOn.getStateWithStorage(moduleName, 1000);
+      expect(JSON.stringify(res)).toContain('requests');
+      await delay(1400);
+      res = await storageAddOn.getStateWithStorage(moduleName, 1000);
+      expect(JSON.stringify(res)).toContain('requests');
+      await delay(500);
+      res = await storageAddOn.getStateWithStorage(moduleName, 1000);
+      expect(JSON.stringify(res)).toEqual('{}');
+      await expect(circuit.fn(failureAsync).execute('dummy')).rejects.toEqual('dummy');
+      expect(slidingCountBreaker.state).toEqual(Mollitia.BreakerState.CLOSED);
+    });
+    it('Should use AddOn storage TTL if found and no module TTL- SCB', async () => {
+      const moduleName = 'mySlidingCountBreaker9';
+      const breakerData = {
+        slidingWindowSize: 3,
+        minimumNumberOfCalls: 2,
+        openStateDelay: 2000,
+        failureRateThreshold: 60,
+        storage: {
+          use: true
+        },
+        name: moduleName
+      };
+      const slidingCountBreaker = new Mollitia.SlidingCountBreaker(breakerData);
+      const slidingCountBreaker2 = new Mollitia.SlidingCountBreaker(breakerData);
+      const circuit = new Mollitia.Circuit({
+        options: {
+          modules: [slidingCountBreaker]
+        }
+      });
+      const circuit2 = new Mollitia.Circuit({
+        options: {
+          modules: [slidingCountBreaker2]
+        }
+      });
+      await slidingCountBreaker.clearState();
+      await expect(circuit.fn(failureAsync).execute('dummy')).rejects.toEqual('dummy');
+      await expect(circuit2.fn(successAsync).execute('dummy')).resolves.toEqual('dummy');
+      expect(slidingCountBreaker2.state).toEqual(Mollitia.BreakerState.CLOSED);
+      await delay(100);
+      let res = await storageAddOn.getStateWithStorage(moduleName, 1000);
+      expect(JSON.stringify(res)).toContain('requests');
+      await delay(1000);
+      res = await storageAddOn.getStateWithStorage(moduleName, 1000);
+      expect(JSON.stringify(res)).toEqual('{}');
+    });
+  });
+},{
+    timeout: 300000,
```

Comment:
Seems like a high value, is this really needed?
Of course we need to migrate to fake timers, so no worries here